.travis.yml

```diff
@@ -10,185 +10,8 @@ python:
-    - "TESTDIR=Java/tapestry"
-    - "TESTDIR=Java/undertow"
-    - "TESTDIR=Java/undertow-jersey-c3p0"
-    - "TESTDIR=Java/undertow-jersey-hikaricp"
-    - "TESTDIR=Java/vertx"
-    - "TESTDIR=Java/vertx-web"
-    - "TESTDIR=Java/wicket"
-    - "TESTDIR=Java/beyondj"
-    - "TESTDIR=Java/wildfly-ee7"
-    - "TESTDIR=JavaScript/express"
-    - "TESTDIR=JavaScript/hapi"
-    - "TESTDIR=JavaScript/koa"
-    - "TESTDIR=JavaScript/nodejs"
-    - "TESTDIR=JavaScript/ringojs"
-    - "TESTDIR=JavaScript/ringojs-convenient"
-    - "TESTDIR=JavaScript/sailsjs"
-    - "TESTDIR=Kotlin/hexagon"
-    - "TESTDIR=Lua/lapis"
-    - "TESTDIR=Lua/octopus"
-    - "TESTDIR=Lua/openresty"
-    - "TESTDIR=Nim/jester"
-    - "TESTDIR=Perl/dancer"
-    - "TESTDIR=Perl/kelp"
-    - "TESTDIR=Perl/mojolicious"
-    - "TESTDIR=Perl/plack"
-    - "TESTDIR=Perl/web-simple"
-    - "TESTDIR=PHP/cakephp"
-    - "TESTDIR=PHP/hhvm"
-    - "TESTDIR=PHP/php"
-    - "TESTDIR=PHP/cygnite"
-    - "TESTDIR=PHP/codeigniter"
-    - "TESTDIR=PHP/clancats"
-    - "TESTDIR=PHP/fat-free"
-    - "TESTDIR=PHP/fuel"
-    - "TESTDIR=PHP/kohana"
-    - "TESTDIR=PHP/laravel"
-    - "TESTDIR=PHP/limonade"
-    - "TESTDIR=PHP/lithium"
-    - "TESTDIR=PHP/lumen"
-    - "TESTDIR=PHP/phalcon"
-    - "TESTDIR=PHP/phalcon-micro"
-    - "TESTDIR=PHP/phpixie"
-    - "TESTDIR=PHP/silex"
-    - "TESTDIR=PHP/silex-orm"
-    - "TESTDIR=PHP/slim"
-    - "TESTDIR=PHP/symfony2"
-    - "TESTDIR=PHP/symfony2-stripped"
-    - "TESTDIR=PHP/yaf"
-    - "TESTDIR=PHP/yii2"
-    - "TESTDIR=PHP/zend"
-    - "TESTDIR=PHP/zend1"
-    - "TESTDIR=PHP/phreeze"
-    - "TESTDIR=Python/asyncio"
-    - "TESTDIR=Python/bottle"
-    - "TESTDIR=Python/cherrypy"
-    - "TESTDIR=Python/django"
-    - "TESTDIR=Python/falcon"
-    - "TESTDIR=Python/flask"
-    - "TESTDIR=Python/historical"
-    - "TESTDIR=Python/klein"
-    - "TESTDIR=Python/pyramid"
-    - "TESTDIR=Python/tornado"
-    - "TESTDIR=Python/turbogears"
-    - "TESTDIR=Python/uwsgi"
-    - "TESTDIR=Python/web2py"
-    - "TESTDIR=Python/weppy"
-    - "TESTDIR=Python/wheezyweb"
-    - "TESTDIR=Python/wsgi"
-    - "TESTDIR=Ruby/grape"
-    - "TESTDIR=Ruby/ngx_mruby"
-    - "TESTDIR=Ruby/padrino"
-    - "TESTDIR=Ruby/rack"
-    - "TESTDIR=Ruby/rack-sequel"
-    - "TESTDIR=Ruby/rails"
-    - "TESTDIR=Ruby/rails-stripped"
-    - "TESTDIR=Ruby/roda-sequel"
-    - "TESTDIR=Ruby/sinatra"
-    - "TESTDIR=Ruby/sinatra-sequel"
-    - "TESTDIR=Rust/iron"
-    - "TESTDIR=Rust/nickel"
-    - "TESTDIR=Rust/hyper"
-    - "TESTDIR=Rust/tokio-minihttp"
-    - "TESTDIR=Scala/akka-http"
-    - "TESTDIR=Scala/blaze"
-    - "TESTDIR=Scala/colossus"
-    - "TESTDIR=Scala/finagle"
-    - "TESTDIR=Scala/finatra"
-    - "TESTDIR=Scala/fintrospect"
-    - "TESTDIR=Scala/play2-scala"
-    - "TESTDIR=Scala/scruffy"
-    - "TESTDIR=Scala/spray"
-    - "TESTDIR=Scala/s-server"
-    - "TESTDIR=Scala/http4s"
-    - "TESTDIR=Scala/finch"
-    - "TESTDIR=Ur/urweb"
+    - "TESTDIR=Java/act"
+
```

Comment:
Make sure to correct the `.travis.yml` file. Note that if you're doing a lot of pushing and you want travis to only run for your framework, I've built a hook - in the commit message you can do `[ci fw-only Java/act]`